.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+name: Run tests
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run install
+        uses: borales/actions-yarn@v4
+        with:
+          cmd: install # will run `yarn install` command
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }} # if needed
```

Comment:
useless because we are not building an open-source library published on https://www.npmjs.com/